src/extension.ts

```diff
@@ -26,7 +26,8 @@ import { activate as activateBBE } from './bbe';
 import { activate as activateTelemetryListener } from './telemetry';
 import { activateDebugConfigProvider } from './debugger';
 import { activate as activateProjectFeatures } from './project';
-import { activate as activateSyntaxHighlighter } from './syntax-highlighter';
+// import { activate as activateSyntaxHighlighter } from './syntax-highlighter';
```

Comment:
If this is the semantic highlighter, shall we get rid of this? Since it is no longer supported and even in earlier versions, this feature is in preview mode. WDYT?

Reply:
+1, removed syntax highliting feature.